src/js/annotations/barplot.js

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
This entire file seems to be copied from [src/js/annotations/histogram.js](https://github.com/eweitz/ideogram/blob/8ecea503da29e98ef6434acfae2981c68023f9c4/src/js/annotations/histogram.js), and pasted here, with two lines added.  (See my comment containing "The only two additions are".)

From that and the information in [your more detailed explanation](https://github.com/eweitz/ideogram/pull/169#issuecomment-531636921) explaining the purpose of this code, I think we can achieve your goal more simply by deleting this file, and incorporating those two lines into `histogram.js`.

That would eliminate a large amount of duplicate code, and also do away with a confusing distinction between histograms and bar plots this PR introduces.